packages/ui/src/utils/utils.ts

```diff
@@ -278,3 +278,26 @@ export function splitObject(
   });
   return [left, right] as const;
 }
+
+export const cloneDeep = function (obj: unknown) {
```

Comment:
[Original file](https://github.com/amzn/style-dictionary/blob/1df04d32161ae34bb3899493a87f7e5f26636acf/lib/utils/es6_.js#L35)
Only change is adding types

Reply:
Why can't this be declared as:
```
const cloneDeep = (obj: unknown) => {
```

Reply:
It can, this is how style-dictionary does it though